CHANGELOG.md

```diff
@@ -21,13 +23,25 @@
 - `az_iot_retry_calc_delay()` renamed to `az_iot_calculate_retry_delay()`.
 - `az_iot_hub_client_sas_get_password()` parameter `token_expiration_epoch_time` moved to second parameter.
 - `az_iot_provisioning_client_init()` parameter `global_device_endpoint` renamed to `global_device_hostname`.
+- `az_iot_provisioning_client_query_status_get_publish_topic()` now accepts the `operation_id` from the `register_response` as the second parameter instead of the whole `az_iot_provisioning_client_register_response` struct.
```

Comment:
Looks like we had missed this change. @ericwol-msft, @danewalton - please take another quick look.

I went through the entire diff to capture any header file changes: https://github.com/Azure/azure-sdk-for-c/compare/c3c272b6bfcc76b78c5736f2eb63abff0a3d1c30...d41afa78bbc6944e17543e6fa5b2d9d8a64258d8

Reply:
lgtm